### ISSUE TITLE
Functions, Procs, Views script fails

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#237]
In GitLab by @moiri on Apr 1, 2020, 10:28

When running the new script creating the upload views the following errors are produced (phpmyadmin):

```
SQL query: Documentation

create view view_uploadTables
as
select t.id as table_id, r.id as row_id, col.id as col_id, t.name as table_name, col.name as col_name, cell.value as value, t.timestamp
from uploadTables t
inner join uploadRows r on (t.id = r.id_uploadTables)
inner join uploadCells cell on (cell.id_uploadRows = r.id)
inner join uploadCols col on (col.id = cell.id_uploadCols)
DELIMITER //

DROP PROCEDURE IF EXISTS get_uploadTable //

CREATE PROCEDURE get_uploadTable( table_id_param INT )
BEGIN  
    SET @@group_concat_max_len = 32000

MySQL said: Documentation
#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DELIMITER //

DROP PROCEDURE IF EXISTS get_uploadTable //

CREATE PROCEDURE get_' at line 8
```

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 1, 2020, 10:30

This was the script which failed:
[fun_pro_views.sql](http://phhum-a209-cp.unibe.ch:10012/SLP/SLP-sleep_coach/uploads/4cf5953d646c93af21ba8b899bebcb2e/fun_pro_views.sql)

**Comment by Kodzhabashev** [comment]
In GitLab by @Kodzhabashev on Apr 1, 2020, 11:10

mentioned in commit 5605c25ddb8467eb7da393939bd1c09b7515b3ec

**Comment by Kodzhabashev** [comment]
In GitLab by @Kodzhabashev on Apr 1, 2020, 11:12

There were missing ; at the end of one of the views. Apparently it worked on Windows and not on Linux (as usual)
Can you send me a private key so I can create an account for you on the test server. I also installed phpmyadmin.

**Comment by Kodzhabashev** [comment]
In GitLab by @Kodzhabashev on Apr 1, 2020, 11:12

After I fix the view, you have to rebuild the big script.

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 1, 2020, 11:15

Cool, thanks.
Would it also be possible for you to create a procedure `get_table_data` similar to `get_form_data`?
I think you will be quicker in doing this.

**Comment by Kodzhabashev** [comment]
In GitLab by @Kodzhabashev on Apr 1, 2020, 11:27

It is already created:
call get_uploadTable(id of the table); we can rename it if you want

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 1, 2020, 12:16

Sorry, my bad.

**Comment by Kodzhabashev** [comment]
In GitLab by @Kodzhabashev on Apr 1, 2020, 12:22

closed via commit 5605c25ddb8467eb7da393939bd1c09b7515b3ec
